packages/react/src/Hero/Hero.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { Hero } from './Hero'
+import '@testing-library/jest-dom'
+
+describe('Hero', () => {
+  it('renders a heading role element', () => {
+    render(<Hero>Breaking news</Hero>)
+
+    const hero = screen.getByRole('heading', {
+      name: 'Breaking news',
+    })
+
+    expect(hero).toBeInTheDocument()
+    expect(hero).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Hero />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('amsterdam-hero')
+  })
+
+  it('can have a custom class name', () => {
```

Comment:
I think maybe this test is a bit redundant. What you're actually testing here is the same as the next test, whether a component can have an additional class. Our original classes are never overwritten. 

I do think we should allow additional classes, otherwise our users can't restyle our components.